web-registry/sanity-codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: 'https://jm12rn9t.api.sanity.io/v1/graphql/staging/slugs'
+schema: 'https://jm12rn9t.api.sanity.io/v1/graphql/staging/default'
```

Comment:
@blushi I want to make sure you see this - I thought because our sanity setup is deploying staging to `/default` that's what we should use, but think I recall you setting it to `/slugs` for a reason - is this okay?

Reply:
yes, keeping this `slugs` tag was an oversight anyway (I had changed that so that we can work on parallel on different sanity changes without interfering with each other), we should use `default` tag by default